### PR TITLE
feat: design for main panel

### LINKED ISSUE #370: Design for MainPanel (tabs and editor and split)

### PR DESCRIPTION
**Pull request by sehyod**
This updates the styles for the main panel, which includes the split bar and the tabs on the top.
This also updates the styles of the RefStudio editor: the menu bar and the text styles inside the editor itself

Before:
image

After:
Screenshot 2023-08-23 at 16 45 42

Closes #370

### CODE REVIEW

File: src/application/components/OpenEditorsTabPane.tsx [hunk 1]
```diff
@@ -18,15 +20,19 @@ export function OpenEditorsTabPane({ paneId }: OpenEditorsTabPaneProps) {
...
-      paneId,
-    } as TabPaneTabContextMenuProps,
-  }));
+  const items = openEditorsData.map((editorData) => {
+    const { type } = parseEditorId(editorData.id);
+    return {
+      text: editorData.title,
+      value: editorData.id,
+      isDirty: editorData.isDirty,
+      Icon: type === 'refstudio' ? <RefStudioEditorIcon /> : undefined,
```
Comment (cguedes) [inline src/application/components/OpenEditorsTabPane.tsx:29]: @sehyod can we have icons for at least PDFs? (please ask the design team)

File: src/components/TabCloseButton.tsx [hunk 2]
```diff
@@ -28,6 +28,7 @@ export function TabCloseButton({ isDirty, onClick }: CloseButtonProps) {
         className={cx('block', 'group-hover/tab-close-button:!block', {
           '!hidden': isDirty,
         })}
+        color="currentcolor"
```
Comment (cguedes) [inline src/components/TabCloseButton.tsx:31]: Why do we need this? Where have you seen this documentation?

File: src/features/textEditor/components/TipTapEditor.css [hunk 3]
```diff
@@ -1,36 +1,40 @@
...
+  @apply normal-case;
 }
 
 .tiptap-editor *::selection {
   @apply bg-transparent;
 }
 
 .tiptap-editor span.selected {
-  background-color: #c4dcf6;
+  @apply bg-[#c4dcf6];
```
Comment (cguedes) [inline src/features/textEditor/components/TipTapEditor.css:39]: Another color to maybe as the design team for a name, right?

**Review by cguedes — CHANGES_REQUESTED** [review]

File: src/components/TabCloseButton.tsx (hunk 2 above)
Comment (sehyod) [reply src/components/TabCloseButton.tsx:31]: We need it to be able to set the color in the parent `div`, in `TabItem`. That's just a regular CSS keyword: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword

File: src/features/textEditor/components/TipTapEditor.css (hunk 3 above)
Comment (sehyod) [reply src/features/textEditor/components/TipTapEditor.css:39]: Yes! I will move it to a variable and ask the design team for which color we should use

**Review by cguedes — APPROVED** [review]

**Pull request merged**